library/ix-dev/community/netbootxyz/templates/_persistence.tpl

```diff
@@ -0,0 +1,40 @@
+{{- define "netboot.persistence" -}}
+persistence:
+  config:
+    enabled: true
+    {{- include "ix.v1.common.app.storageOptions" (dict "storage" .Values.netbootStorage.config) | nindent 4 }}
+    targetSelector:
+      netboot:
+        netboot:
+          mountPath: /config
+        {{- if and (eq .Values.netbootStorage.config.type "ixVolume")
+                  (not (.Values.netbootStorage.config.ixVolumeConfig | default dict).aclEnable) }}
+        01-permissions:
+          mountPath: /mnt/directories/config
+        {{- end }}
```

Comment:
Lets remove the permissions sections, as it does it itself on the startup
https://github.com/netbootxyz/docker-netbootxyz/blob/2a36c1a8e06a6dd79dca0bbc6019e097ae46cbc6/root/start.sh#L18-L21